docs/guide/testrunner/pageobjects.md

```diff
@@ -116,6 +158,34 @@ describe('login form', function () {
     });
 });
 ```
+Or, when using the ES6 class implementation:
```

Comment:
OK yes with the other alteration this is now not needed and can be just the one so removed this section